File: src/main/java/org/asmolinski/java/Nullability.java
package org.asmolinski.java;

public class Nullability {

    public static void main(String[] args) {
        System.out.println(stringFun().length());

        if (booleanFun()) {
            System.out.println("prawda");
        } else {
            System.out.println("fałsz");
        }
    }

    private static Boolean booleanFun() {
        return null;
    }

    private static String stringFun() {
        return null;
    }
}

File: src/main/kotlin/org/asmolinski/kotlin/Nullability.kt
package org.asmolinski.kotlin

fun main() {
    println(stringFun()?.length)

    if (booleanFun() == true) {
        println("prawda")
    } else {
        println("fałsz")
    }
}

private fun booleanFun(): Boolean? {
    return null
}

private fun stringFun(): String? {
    return null
}